### ISSUE TITLE
Allow only https URLs.

### ISSUE THREAD

**Issue by samgiles** [mozilla-magnet/magnet-client#90]
We shouldn't even hit the network for non-https URLs.

**Comment by wilsonpage** [comment]
I agree long-term that this should be an eventual goal, but I'm concerned that if added too early, we may be shooting ourselves in the foot. Currently the Physical Web ecosystem is severely content poor and we should at first try to deliver anything we can to delight the user.
